### PR TITLE
Eliminated wait-for-keypress after error msg display

### PR DESCRIPTION
**Pull request by chipbarnaby**
## Description

Dropped user prompt after error messages that allowed entering 'A' for abort, any other key to continue.

Now most messages display and execution continues.  Repeated messages will stop execution due to errorCount limit.

In addition, many comments on error reporting calls have been modified to remove the "wait for keypress" (or similar) in the wording.

Updated description of command line options in documentation.  Added description for missing options.

Reviewed documentation to see if keypress behavior is described.  Apparently not.  No changes made.

No test results changes.

## Author Progress Checklist:

- [ ] Open draft pull request
    - [ ] Make title clearly understandable in a standalone change log context
    - [ ] Assign yourself the issue
    - [ ] Add at least one label (enhancement, bug, or maintenance)
    - [ ] Link the issue(s) addressed by this PR (under "Development" in the sidebar menu) 
- [ ] Make code changes (if you haven't already)
- [ ] Self-review of code
    - [ ] My code follows the style guidelines of this project
    - [ ] I have added comments to my code, particularly in hard-to-understand areas
    - [ ] I have only committed the necessary changes for this fix or feature
    - [ ] I have made corresponding changes to the documentation
    - [ ] My changes generate no new warnings
    - [ ] I have ensured that my fix is effective or that my feature works as intended by:
        - [ ] exercising the code changes in the test framework, and/or
        - [ ] manually verifying the changes (as explained in the the pull request description above)
    - [ ] My changes pass all local tests
    - [ ] My changes successfully passes CI checks
    - [ ] Add any unit test for proof and documentation.
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
- [ ] Move pull request out of draft mode and assign reviewers
- [ ] Iterate with reviewers until all changes are approved
    - [ ] Make changes in response to reviewer comments
    - [ ] Merge in main branch and address resulting conflicts and/or test failures.
    - [ ] Re-request review in GitHub

## Reviewer Checklist:

 - [ ] Read the pull request description
 - [ ] Perform a code review on GitHub
 - [ ] Confirm all CI checks pass and there are no build warnings
 - [ ] Pull, build, and run automated tests locally
 - [ ] Perform manual tests of the fix or feature locally
 - [ ] Add any review comments, if applicable
 - [ ] Submit review in GitHub as either
     - [ ] Request changes, or
     - [ ] Approve
 - [ ] Iterate with author until all changes are approved

### CODE REVIEW

File: src/rmkerr.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 1997-2019 The CSE Authors. All rights reserved.
+// Copyright (c) 1997-2023 The CSE Authors. All rights reserved.
```
Comment (nealkruis) [inline src/rmkerr.cpp:1]: This should really be a separate PR.

File: src/msgtbl.cpp [hunk 2]
```diff
@@ -37,7 +37,7 @@ MSGTBL msgTbl[] =		//(far rob 1-92 for BC)
 		"     -ipath;path;...            input & include file drives/directories\n"
 		"     -Dname  or  -Dname=value   define preprocessor symbol\n"
 		"     -Uname                     undefine preprocessor symbol\n"
-		"     -b     batch: don't await keypress at error\n"
+		"     -b     batch: never wait for user input at error\n"
```
[the remark this answers is not in the document]
Comment (chipbarnaby) [reply src/msgtbl.cpp:40]: I decided to leave it since some interactive feature(s) might resurrect.  Perhaps alter the documentation noting it currently has no effect.

[the remark this answers is not in the document]
Comment (nealkruis) [reply src/msgtbl.cpp:40]: I think we should change this to "never prompt for user input (currently no such prompts)"

**Review by nealkruis — APPROVED** [review]
